### PR TITLE
feat(imagery): import titles from job.json

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: config/tileset/aerial.json [hunk 1]
```diff
@@ -39,7 +39,6 @@
     { "2193": "im_01F6P1V84P2HJ1YB0D5X1CSWJH", "3857": "im_01ED83DC7MCMBCRNN45E07HCHH", "name": "tasman_rural_2016-17_0-3m", "minZoom": 13 },
     { "2193": "im_01FB82NZY0ASDQBKT28DSXS8XC", "3857": "im_01FB82QKPWEHV0FWN7KBXM6NDM", "name": "wellington_rural_2021_0-3m_RGB", "minZoom": 13 },
     { "2193": "im_01F6P220GYWC836YQ4RRZGSV73", "3857": "im_01ED83VT2T06B1FVXF9P6CCV8C", "name": "west-coast_rural_2016-17_0-3m", "minZoom": 13 },
-    { "2193": "im_01F6P15FS34RD0KX1FQCQF1CA8", "3857": "im_01ECTS06BSSHA626JHSDTNSC97", "name": "gisborne_rural_2017-18_0-3m", "minZoom": 13 },
```
Comment (Wentao-Kuang) [inline config/tileset/aerial.json:42]: Is this one removed on purpose?

File: config/imagery/hawkes-bay_rural_2018-2019_0-3m_RGB-NZTM2000Quad.json [hunk 2]
```diff
@@ -1,682 +0,0 @@
-{
```
Comment (Wentao-Kuang) [inline config/imagery/hawkes-bay_rural_2018-2019_0-3m_RGB-NZTM2000Quad.json:1]: Unused imagery config?

File: config/imagery/gisborne_rural_2017-18_0-3m-NZTM2000Quad.json [hunk 3]
```diff
@@ -1,332 +0,0 @@
-{
-  "name": "gisborne_rural_2017-18_0-3m",
```
Comment (Wentao-Kuang) [inline config/imagery/gisborne_rural_2017-18_0-3m-NZTM2000Quad.json:2]: Unused gisborne imagery config?

File: config/tileset/aerial.json (hunk 1 above)
Comment (blacha) [reply config/tileset/aerial.json:42]: there is two commits that reference this, maybe i should cherry pick them into their own PR

File: config/imagery/bay-of-plenty_rural_2021_0-2m_RGB-NZTM2000Quad.json [hunk 4]
```diff
@@ -887,5 +887,10 @@
       "y": 5790818.845461301,
       "height": 9783.939620502528
     }
-  ]
+  ],
+  "title": "Bay of Plenty 0.2m  Rural Aerial Photos (2021)",
```
Comment (dwsilk) [inline config/imagery/bay-of-plenty_rural_2021_0-2m_RGB-NZTM2000Quad.json:891]: Double space

File: config/imagery/bay-of-plenty_rural_2021_0-2m_RGB-WebMercatorQuad.json [hunk 5]
```diff
@@ -1216,5 +1216,10 @@
       "y": -4706074.957461745,
       "height": 9783.939620502548
     }
-  ]
+  ],
+  "title": "Bay of Plenty 0.2m  Rural Aerial Photos (2021)",
```
Comment (dwsilk) [inline config/imagery/bay-of-plenty_rural_2021_0-2m_RGB-WebMercatorQuad.json:1220]: Double space

File: config/imagery/central-hawkes-bay_urban_2017-18_0-1m-NZTM2000Quad.json [hunk 6]
```diff
@@ -419,5 +419,10 @@
       "height": 4891.969810251264,
       "name": "11-1060-999"
     }
-  ]
+  ],
+  "title": "Central Hawkes Bay 0.1m Urban Aerial Photos (2017-2018)",
```
Comment (dwsilk) [inline config/imagery/central-hawkes-bay_urban_2017-18_0-1m-NZTM2000Quad.json:423]: Central Hawkes Bay 0.1m Urban Aerial Photos (2017-2018)

https://gazetteer.linz.govt.nz/place/14976

File: config/imagery/central-hawkes-bay_urban_2017-18_0-1m-WebMercatorQuad.json [hunk 7]
```diff
@@ -488,5 +488,10 @@
       "width": 4891.969810251274,
       "height": 4891.969810251274
     }
-  ]
+  ],
+  "title": "Central Hawkes Bay 0.1m Urban Aerial Photos (2017-2018)",
```
Comment (dwsilk) [inline config/imagery/central-hawkes-bay_urban_2017-18_0-1m-WebMercatorQuad.json:492]: Central Hawkes Bay 0.1m Urban Aerial Photos (2017-2018)

https://gazetteer.linz.govt.nz/place/14976

File: config/imagery/chatham-islands_digital-globe_2014-2019_0-5m-NZTM2000Quad-Z12.json [hunk 8]
```diff
@@ -209,5 +209,6 @@
       "height": 19567.879241005056,
       "name": "9-291-272"
     }
-  ]
+  ],
+  "title": "Chatham-islands digital-globe 2014-2019 0.5m"
```
Comment (dwsilk) [inline config/imagery/chatham-islands_digital-globe_2014-2019_0-5m-NZTM2000Quad-Z12.json:213]: Chatham Islands 0.5m Satellite Imagery (2014-2019)

https://data.linz.govt.nz/layer/104485-chatham-islands-05m-satellite-imagery-2014-2019/
https://gazetteer.linz.govt.nz/place/14963

File: config/imagery/chatham-islands_digital-globe_2014-2019_0-5m-NZTM2000Quad-z13.json [hunk 9]
```diff
@@ -216,5 +216,6 @@
       "height": 19567.879241005056,
       "name": "9-292-272"
     }
-  ]
+  ],
+  "title": "Chatham-islands digital-globe 2014-2019 0.5m"
```
Comment (dwsilk) [inline config/imagery/chatham-islands_digital-globe_2014-2019_0-5m-NZTM2000Quad-z13.json:220]: Chatham Islands 0.5m Satellite Imagery (2014-2019)

https://data.linz.govt.nz/layer/104485-chatham-islands-05m-satellite-imagery-2014-2019/
https://gazetteer.linz.govt.nz/place/14963

File: config/imagery/gebco_2020_Nztm2000Quad_305-75m-NZTM2000Quad.json [hunk 10]
```diff
@@ -20,5 +20,6 @@
       "height": 10018754.171400001,
       "name": "0-0-0"
     }
-  ]
+  ],
+  "title": "Gebco 2020.nc"
```
Comment (dwsilk) [inline config/imagery/gebco_2020_Nztm2000Quad_305-75m-NZTM2000Quad.json:24]: GEBCO Gridded Bathymetry (2020)

I'm not exactly sure what the source data is for this and what it should be called, is it https://www.gebco.net/data_and_products/gridded_bathymetry_data/gebco_2020/ ?

File: config/imagery/geographx_nz_dem_2012_8-0m-WebMercatorQuad.json [hunk 11]
```diff
@@ -146,5 +146,6 @@
       "height": 313086.06785608135,
       "name": "7-127-78"
     }
-  ]
+  ],
+  "title": "Geographx nz dem 2012 8.0m"
```
Comment (dwsilk) [inline config/imagery/geographx_nz_dem_2012_8-0m-WebMercatorQuad.json:150]: NZ 8m Digital Elevation Model (2012)

https://data.linz.govt.nz/layer/51768-nz-8m-digital-elevation-model-2012/

File: config/imagery/geographx_nz_texture_shade_2012_8-0m-NZTM2000Quad.json [hunk 12]
```diff
@@ -216,5 +216,6 @@
       "height": 313086.0678560809,
       "name": "5-16-14"
     }
-  ]
+  ],
+  "title": "Geographx nz texture shade 2012 8.0m"
```
Comment (dwsilk) [inline config/imagery/geographx_nz_texture_shade_2012_8-0m-NZTM2000Quad.json:220]: NZ 8m Texture Shade (2012)

Derived from https://data.linz.govt.nz/layer/51768-nz-8m-digital-elevation-model-2012/

File: config/imagery/geographx_nz_texture_shade_2012_8-0m-WebMercatorQuad.json [hunk 13]
```diff
@@ -314,5 +314,6 @@
       "height": 313086.06785608135,
       "name": "7-127-77"
     }
-  ]
+  ],
+  "title": "Geographx nz texture shade 2012 8.0m"
```
Comment (dwsilk) [inline config/imagery/geographx_nz_texture_shade_2012_8-0m-WebMercatorQuad.json:318]: NZ 8m Texture Shade (2012)

Derived from https://data.linz.govt.nz/layer/51768-nz-8m-digital-elevation-model-2012/

File: config/imagery/hawkes-bay_rural_2014-2015_0-30m_RGBA-NZTM2000Quad.json [hunk 14]
```diff
@@ -678,5 +678,10 @@
       "height": 19567.879241005056,
       "name": "9-270-244"
     }
-  ]
+  ],
+  "title": "Hawkes Bay 0.3m Rural Aerial Photos (2014-2015)",
```
Comment (dwsilk) [inline config/imagery/hawkes-bay_rural_2014-2015_0-30m_RGBA-NZTM2000Quad.json:682]: Hawke's Bay 0.3m Rural Aerial Photos (2014-2015)

https://gazetteer.linz.govt.nz/place/14976

File: config/imagery/hawkes-bay_rural_2014-2015_0-30m_RGBA-WebMercatorQuad.json [hunk 15]
```diff
@@ -859,5 +859,10 @@
       "width": 19567.879241005096,
       "height": 19567.879241005096
     }
-  ]
+  ],
+  "title": "Hawkes Bay 0.3m Rural Aerial Photos (2014-2015)",
```
Comment (dwsilk) [inline config/imagery/hawkes-bay_rural_2014-2015_0-30m_RGBA-WebMercatorQuad.json:863]: Hawke's Bay 0.3m Rural Aerial Photos (2014-2015)

https://gazetteer.linz.govt.nz/place/14976

File: config/imagery/hawkes-bay_rural_2019-2020_0.3m_RGB-NZTM2000Quad.json [hunk 16]
```diff
@@ -687,5 +687,10 @@
       "x": 2022740.666671365,
       "y": 5644059.7511537615
     }
-  ]
+  ],
+  "title": "Hawkes Bay 0.3m Rural Aerial Photos (2019-2020)",
```
Comment (dwsilk) [inline config/imagery/hawkes-bay_rural_2019-2020_0.3m_RGB-NZTM2000Quad.json:691]: Hawke's Bay 0.3m Rural Aerial Photos (2019-2020)

https://gazetteer.linz.govt.nz/place/14976

File: config/imagery/hawkes-bay_rural_2019-2020_0.3m_RGB-WebMercatorQuad.json [hunk 17]
```diff
@@ -918,5 +918,10 @@
       "x": 19802693.791897174,
       "y": -4754994.655564257
     }
-  ]
+  ],
+  "title": "Hawkes Bay 0.3m Rural Aerial Photos (2019-2020)",
```
Comment (dwsilk) [inline config/imagery/hawkes-bay_rural_2019-2020_0.3m_RGB-WebMercatorQuad.json:922]: Hawke's Bay 0.3m Rural Aerial Photos (2019-2020)

https://gazetteer.linz.govt.nz/place/14976

File: config/imagery/hutt-city_urban_2017_0-10m-NZTM2000Quad.json [hunk 18]
```diff
@@ -153,5 +153,10 @@
       "height": 4891.969810251264,
       "name": "11-1027-1021"
     }
-  ]
+  ],
+  "title": "Hutt City 0.10m Urban Aerial Photos (2017)",
```
Comment (dwsilk) [inline config/imagery/hutt-city_urban_2017_0-10m-NZTM2000Quad.json:157]: Lower Hutt 0.10m Urban Aerial Photos (2017)

https://gazetteer.linz.govt.nz/place/26940

File: config/imagery/hutt-city_urban_2017_0-10m-WebMercatorQuad.json [hunk 19]
```diff
@@ -271,5 +271,10 @@
       "width": 4891.969810251274,
       "height": 4891.969810251274
     }
-  ]
+  ],
+  "title": "Hutt City 0.10m Urban Aerial Photos (2017)",
```
Comment (dwsilk) [inline config/imagery/hutt-city_urban_2017_0-10m-WebMercatorQuad.json:275]: Lower Hutt 0.10m Urban Aerial Photos (2017)

https://gazetteer.linz.govt.nz/place/26940

File: config/imagery/hutt-city_urban_2021_0-075m_RGB-NZTM2000Quad.json [hunk 20]
```diff
@@ -153,5 +153,10 @@
       "y": 5438597.019123209,
       "height": 4891.969810251264
     }
-  ]
+  ],
+  "title": "Hutt City 0.075m Urban Aerial Photos (2021)",
```
Comment (dwsilk) [inline config/imagery/hutt-city_urban_2021_0-075m_RGB-NZTM2000Quad.json:157]: Lower Hutt 0.075m Urban Aerial Photos (2021)

https://gazetteer.linz.govt.nz/place/26940

File: config/imagery/hutt-city_urban_2021_0-075m_RGB-WebMercatorQuad.json [hunk 21]
```diff
@@ -272,5 +272,10 @@
       "y": -5038728.90455883,
       "height": 4891.969810251274
     }
-  ]
+  ],
+  "title": "Hutt City 0.075m Urban Aerial Photos (2021)",
```
Comment (dwsilk) [inline config/imagery/hutt-city_urban_2021_0-075m_RGB-WebMercatorQuad.json:276]: Lower Hutt 0.075m Urban Aerial Photos (2021)

https://gazetteer.linz.govt.nz/place/26940

File: config/imagery/kaikoura-post-earthquake_rural_2016-17_0-3m-NZTM2000Quad.json [hunk 22]
```diff
@@ -496,5 +496,10 @@
       "height": 19567.879241005056,
       "name": "9-253-258"
     }
-  ]
+  ],
+  "title": "Kaikoura 0.3m Rural Aerial Photos (2016-2017)",
```
Comment (dwsilk) [inline config/imagery/kaikoura-post-earthquake_rural_2016-17_0-3m-NZTM2000Quad.json:500]: Kaikōura 0.3m Rural Aerial Photos (2016-2017)

https://gazetteer.linz.govt.nz/place/24119

File: config/imagery/kaikoura-post-earthquake_rural_2016-17_0-3m-WebMercatorQuad.json [hunk 23]
```diff
@@ -663,5 +663,10 @@
       "width": 19567.879241005096,
       "height": 19567.879241005096
     }
-  ]
+  ],
+  "title": "Kaikoura 0.3m Rural Aerial Photos (2016-2017)",
```
Comment (dwsilk) [inline config/imagery/kaikoura-post-earthquake_rural_2016-17_0-3m-WebMercatorQuad.json:667]: Kaikōura 0.3m Rural Aerial Photos (2016-2017)

https://gazetteer.linz.govt.nz/place/24119

File: config/imagery/manawatu-whanganui_rural_2015-16_0-3m-WebMercatorQuad.json [hunk 24]
```diff
@@ -817,5 +817,10 @@
       "width": 19567.879241005096,
       "height": 19567.879241005096
     }
-  ]
+  ],
+  "title": "Manawatu Whanganui 0.3m Rural Aerial Photos (2015-2016)",
```
Comment (dwsilk) [inline config/imagery/manawatu-whanganui_rural_2015-16_0-3m-WebMercatorQuad.json:821]: Manawatū-Whanganui 0.3m Rural Aerial Photos (2015-2016)

https://gazetteer.linz.govt.nz/place/58038

File: config/imagery/manawatu-whanganui_rural_2015-16_0-3m-NZTM2000Quad.json [hunk 25]
```diff
@@ -692,5 +692,10 @@
       "height": 19567.879241005056,
       "name": "9-263-251"
     }
-  ]
+  ],
+  "title": "Manawatu Whanganui 0.3m Rural Aerial Photos (2015-2016)",
```
Comment (dwsilk) [inline config/imagery/manawatu-whanganui_rural_2015-16_0-3m-NZTM2000Quad.json:696]: Manawatū-Whanganui 0.3m Rural Aerial Photos (2015-2016)

https://gazetteer.linz.govt.nz/place/58038

File: config/imagery/manawatu-whanganui_rural_2016-17_0-3m-NZTM2000Quad.json [hunk 26]
```diff
@@ -433,5 +433,10 @@
       "height": 19567.879241005056,
       "name": "9-259-240"
     }
-  ]
+  ],
+  "title": "Manawatu Whanganui 0.3m Rural Aerial Photos (2016-2017)",
```
Comment (dwsilk) [inline config/imagery/manawatu-whanganui_rural_2016-17_0-3m-NZTM2000Quad.json:437]: Manawatū-Whanganui 0.3m Rural Aerial Photos (2016-2017)

https://gazetteer.linz.govt.nz/place/58038

File: config/imagery/manawatu-whanganui_rural_2016-17_0-3m-WebMercatorQuad.json [hunk 27]
```diff
@@ -467,5 +467,10 @@
       "width": 19567.879241005096,
       "height": 19567.879241005096
     }
-  ]
+  ],
+  "title": "Manawatu Whanganui 0.3m Rural Aerial Photos (2016-2017)",
```
Comment (dwsilk) [inline config/imagery/manawatu-whanganui_rural_2016-17_0-3m-WebMercatorQuad.json:471]: Manawatū-Whanganui 0.3m Rural Aerial Photos (2016-2017)

https://gazetteer.linz.govt.nz/place/58038

File: config/imagery/new_zealand_sentinel_2019-2020_10m-NZTM2000Quad.json [hunk 28]
```diff
@@ -132,5 +132,6 @@
       "height": 626172.1357121618,
       "name": "4-8-7"
     }
-  ]
+  ],
+  "title": "New zealand sentinel 2019-2020 10m"
```
Comment (dwsilk) [inline config/imagery/new_zealand_sentinel_2019-2020_10m-NZTM2000Quad.json:136]: NZ 10m Satellite Imagery (2019-2020)

https://data.linz.govt.nz/layer/104735-nz-10m-satellite-imagery-2019-2020/

File: config/imagery/new_zealand_sentinel_2019-2020_10m-WebMercatorQuad.json [hunk 29]
```diff
@@ -110,5 +110,6 @@
       "width": 626172.1357121634,
       "height": 626172.1357121634
     }
-  ]
+  ],
+  "title": "New zealand sentinel 2019-2020 10m"
```
Comment (dwsilk) [inline config/imagery/new_zealand_sentinel_2019-2020_10m-WebMercatorQuad.json:114]: NZ 10m Satellite Imagery (2019-2020)

https://data.linz.govt.nz/layer/104735-nz-10m-satellite-imagery-2019-2020/

File: config/imagery/nz_satellite_2020-2021_10m_RGB-NZTM2000Quad.json [hunk 30]
```diff
@@ -132,5 +132,6 @@
       "y": 5428813.079502706,
       "height": 626172.1357121618
     }
-  ]
+  ],
+  "title": "NZ 10m Satellite Imagery (2020-2021)"
```
Comment (dwsilk) [inline config/imagery/nz_satellite_2020-2021_10m_RGB-NZTM2000Quad.json:136]: NZ 10m Satellite Imagery (2020-2021)

https://data.linz.govt.nz/layer/106279-nz-10m-satellite-imagery-2020-2021/

File: config/imagery/nz_satellite_2020-2021_10m_RGB-WebMercatorQuad.json [hunk 31]
```diff
@@ -111,5 +111,6 @@
       "y": -5009377.085697336,
       "height": 626172.1357121634
     }
-  ]
+  ],
+  "title": "NZ 10m Satellite Imagery (2020-2021)"
```
Comment (dwsilk) [inline config/imagery/nz_satellite_2020-2021_10m_RGB-WebMercatorQuad.json:115]: NZ 10m Satellite Imagery (2020-2021)

https://data.linz.govt.nz/layer/106279-nz-10m-satellite-imagery-2020-2021/

**Review by dwsilk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**